lms/static/scripts/frontend_apps/services/test/content-info-fetcher-test.js

```diff
@@ -42,12 +55,21 @@ describe('ContentInfoFetcher', () => {
           title: 'JSTOR homepage',
           link: 'https://www.jstor.org',
         },
+        container: {
+          title: 'Test container',
+          subtitle: 'Test container subtitle',
+        },
         item: {
           title: 'Test article',
-          containerTitle: '',
+          subtitle: 'Test article subtitle',
+          containerTitle: 'Test container',
         },
 
-        links: {},
+        links: {
+          currentItem: 'https://www.jstor.org/stable/123456',
+          nextItem: 'https://www.jstor.org/stable/333333',
+          previousItem: 'https://www.jstor.org/stable/111111',
+        },
```

Comment:
This does look like something we should maybe emit.

Reply:
Sure, if you'd like to add a `links` dict to the response I will consume it (in a later PR)!